demisto_sdk/commands/pre_commit/pre_commit_command.py

```diff
@@ -138,61 +139,83 @@ def run(
             str(path) for path in sorted(PYTHONPATH) if "site-packages" not in str(path)
         )
         precommit_env["DEMISTO_SDK_CONTENT_PATH"] = str(CONTENT_PATH)
-        for python_version, changed_files in self.python_version_to_files.items():
-            precommit_config = deepcopy(self.precommit_template)
-            assert isinstance(precommit_config, dict)
-            changed_files_string = ", ".join(
-                sorted((str(changed_path) for changed_path in changed_files))
-            )
-            logger.info(
-                f"Running pre-commit with Python {python_version} on {changed_files_string}"
-            )
-            if python_version.startswith("2"):
+        if changed_files_python2 := self.python_version_to_files.pop(
```

Comment:
I don't really see a point for this code now.
We can delete it and skip adding python2 files in `files` key in each `prepare_hook`.